scripts/dev

```diff
@@ -10,12 +10,6 @@ if [[ ! -d "${PWD}/config" ]]; then
     hass --config "${PWD}/config" --script ensure_config
 fi
 
-# Set the path to custom_components
-## This let's us have the structure we want <root>/custom_components/integration_blueprint
-## while at the same time have Home Assistant configuration inside <root>/config
-## without resulting to symlinks.
-export PYTHONPATH="${PYTHONPATH}:${PWD}/custom_components"
```

Comment:
Why was this removed?

Reply:
No longer needed since custom_components is mounted in devcontainer.json. 

Pythonpath won’t work anymore unless you remove dependency on nordpool package due to name collision.

Reply:
I have already removed the nordpool dependency. Please add it back so the scripts still works without dev container.